### ISSUE TITLE
How can I get the probability of a word which fits the masked place?

### ISSUE THREAD

**Issue by traymihael** [huggingface/transformers#1611]
## ❓ Questions & Help

I want to get the probability of a word which fits the masked place.
```
tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
model = BertForMaskedLM.from_pretrained('bert-base-uncased')
model.eval()

text = '[CLS] I want to [MASK] the car because it is cheap . [SEP]'
tokenized_text = tokenizer.tokenize(text)
indexed_tokens = tokenizer.convert_tokens_to_ids(tokenized_text)
segments_ids = [0] * len(tokenized_text)
tokens_tensor = torch.tensor([indexed_tokens])
segments_tensors = torch.tensor([segments_ids])

with torch.no_grad():
    predictions = model(tokens_tensor, segments_tensors)

masked_index = tokenized_text.index('[MASK]')
predicted_score, predicted_indexes = torch.topk(predictions[0][0, masked_index], k=5)
predicted_tokens = tokenizer.convert_ids_to_tokens(predicted_indexes.tolist())
```
`predicted_tokens`
> `['buy', 'sell', 'rent', 'take', 'drive']`

'predicted_score'
> 'tensor([10.9675, 10.4480,  9.5352,  9.5170,  9.3046])'

`predicted_score` is not a probability. I want a pair of the word and its probability (the total score for all words is 1) which goes in the [MASK].
